base/REPL.jl

```diff
@@ -416,7 +416,7 @@ function add_history(hist::REPLHistoryProvider, s)
     push!(hist.history, str)
     hist.history_file === nothing && return
     entry = """
-    # time: $(Libc.strftime("%Y-%m-%d %H:%M:%S %Z", time()))
+    # time: $( string( now() ) )
```

Comment:
I like the spaces. They make it easier for me to read code. I can change it if no one else agrees. Is string unnecessary because $ will do it implicitly?